src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -72,7 +72,7 @@ class MockWorkflowManagerActor extends Actor  {
       }
```

Comment:
Although the tests here probably fake out too much to validate the correctness of the scatter handling for the `logs` endpoints, it might be nice to have a test in here anyway just to document what scatter responses are supposed to look like since AFAICT that's not documented anywhere else.